components/KeyFinancials.vue

```diff
@@ -41,17 +41,33 @@
     mixins: [Global],
 
     props: {
-      'content': Array,
-      'ftsUrl': String,
+      'ftsRawData': {
+        required: false,
+        type: Array,
+        default: () => [],
+      },
+      'ftsUrl': {
+        required: false,
+        type: String,
+        default: '',
+      },
+      'ftsManualData': {
+        required: false,
+        type: Array,
+        default: () => [],
+      }
     },
 
     computed: {
       cssId() {
-        if (typeof this.content === 'Array' && this.content.length > 0) {
-          return 'cf-' + this.content.map((item) => item.sys.id).join('_');
+        if (this.ftsPlanId) {
+          return 'cf-keyFinancials-fts-' + this.ftsPlanId;
```

Comment:
this addition will also pave the way for if we were provide more than one FTS box. By using the plan ID in the `cssId` we can target boxes individually during PNG Snaps.